Tryouts/Messaging-JS/package.json

```diff
@@ -1,25 +1,36 @@
 {
-    "name": "@morgan-stanley/compose-messaging-client",
+    "name": "@morgan-stanley/composeui-messaging-client",
     "version": "0.1.0",
     "description": "JavaScript client for Compose UI's Message Router",
-    "main": "index.js",
+    "type": "module",
+    "files": [
+        "dist/esm/**/*"
+    ],
+    "main": "dist/esm/index.js",
+    "module": "dist/esm/index.js",
+    "types": "dist/esm/index.d.ts",
     "scripts": {
-        "build": "yarn run build:clean && yarn run build:esm && yarn run build:cjs",
+        "build": "yarn && yarn run build:clean && yarn run build:rollup",
```

Comment:
if you want to use yarn then it should be

- a) added as a devDependency
- b) invoked with npx

Unless you do that everyone in the dev team has to globally install yarn first

Reply:
Understood, removing the dependency on yarn

Reply:
ok. So delete yar.lock from Messaging-JS
and the build script should look like:  "build": "npm run build:clean && npm run build:rollup",